Summary.md

```diff
@@ -1,5 +1,20 @@
 # Summary
 
-* [New introduction](/docs/Readme.md)
+* [Quickstart](/docs/Readme.md)
 * [Contributing]()
-    * [Docs](/docs/contributing/Docs.md)
\ No newline at end of file
+    * [Installation](/docs/contributing/Installation.md)
+    * [Creating a new application](/docs/contributing/CreatingNewApp.md)
```

Comment:
Please put this and all the other documents you added inside `development` instead of `contributing` (that will be converted to a single file, see above). We will update that anyway, but this is just for now.